src/Arcus.Testing.Tests.Integration/Arcus.Testing.Tests.Integration.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
Let's bump to 3.1 since 3.0 is already end of life

Reply:
Ok!